### PR TITLE
Alchemy Rebalancing for January 12, 2024

### PR DESCRIPTION
**Pull request by paulsnoops**
Source: https://magic.wizards.com/en/news/mtg-arena/mtg-arena-announcements-january-12-2024

- [x] Briar Hydra
- [x] Excavation Explosion
- [x] Geology Enthusiast
- [x] Hall of Tagsin
- [x] Hurkyl's Prodigy (Alchemy only - no new script needed)
- [x] Karn, Living Legacy
- [x] Llanowar Greenwidow
- [x] Llanowar Loamspeaker
- [x] Meria's Outrider
- [x] Mightstone's Animation
- [x] Mishra, Excavation Prodigy
- [x] Nael, Avizoa Aeronaut
- [x] Radha's Firebrand
- [x] Radha, Coalition Warlord
- [x] Rulik Mons, Warren Chief
- [x] Saheeli, Filigree Master
- [x] Scout the Wilderness
- [x] Soul of Windgrace
- [x] Splitting the Powerstone
- [x] Sprouting Goblin
- [x] Sunbathing Rootwalla
- [x] Tatyova, Steward of Tides
- [x] Thran Portal
- [x] Thran Spider
- [x] Urza's Command
- [x] Urza, Powerstone Prodigy
- [x] Uurg, Spawn of Turg
- [x] Visions of Phyrexia
- [x] Yotian Courier (Alchemy only - no new script needed)
- [x] Zar Ojanen, Scion of Efrava

### CODE REVIEW

**Review by tool4ever — DISMISSED** [review]

File: forge-gui/res/cardsfolder/r/radhas_firebrand.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ ManaCost:1 R
 Types:Creature Human Warrior
 PT:3/1
 T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigPump | TriggerDescription$ Whenever CARDNAME attacks, target creature defending player controls with power less than CARDNAME's power can't block this turn.
-SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.DefenderCtrl+powerLTX | TgtPrompt$ Select target creature an opponent controls with power less than than Radha's Firebrand's power | KW$ HIDDEN CARDNAME can't block. | IsCurse$ True
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.DefenderCtrl+powerLTX | TgtPrompt$ Select target creature an opponent controls with power less than Radha's Firebrand's power | KW$ HIDDEN CARDNAME can't block. | IsCurse$ True
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/r/radhas_firebrand.txt:6]: in TgtPrompt should probably replace `Radha's Firebrand` with `CARDNAME`

File: forge-gui/res/cardsfolder/rebalanced/a-hall_of_tagsin.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:A-Hall of Tagsin
+ManaCost:no cost
+Types:Land
+A:AB$ Mana | Cost$ T | Produced$ C | SpellDescription$ Add {C}.
+A:AB$ Mana | Cost$ 1 T | Produced$ Any | SpellDescription$ Add one mana of any color.
+A:AB$ Token | Cost$ 3 T | TokenAmount$ 1 | TokenTapped$ True | TokenScript$ c_a_powerstone | SpellDescription$ Create a tapped Powerstone token.
+DeckHas:Ability$Token & Type$Artifact
+Oracle:{T}: Add {C}.\n{1}, {T}: Add one mana of any color.\n{3}, {T}, Create a tapped Powerstone token.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/rebalanced/a-hall_of_tagsin.txt:8]: should be `{3}, {T}:`
Typo in original script as well

File: forge-gui/res/cardsfolder/rebalanced/a-saheeli_filigree_master.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+A:AB$ Token | Cost$ SubCounter<2/LOYALTY> | Planeswalker$ True | PumpDuration$ EOT | TokenAmount$ 2 | PumpKeywords$ Haste | TokenScript$ c_1_1_a_thopter_flying | TokenOwner$ You | SpellDescription$ Create two 1/1 colorless Thopter artifact creature tokens with flying. They gain haste until end of turn.
+A:AB$ Effect | Cost$ SubCounter<4/LOYALTY> | Name$ Emblem - Saheeli, Filigree Master | StaticAbilities$ EmblemArtifactPump,ReduceCost | Planeswalker$ True | Ultimate$ True | Stackable$ False | Duration$ Permanent | AILogic$ Always | SpellDescription$ You get an emblem with "Artifact creatures you control get +1/+1" and "Artifact spells you cast cost 1 less to cast."
+SVar:EmblemArtifactPump:Mode$ Continuous | AddPower$ 1 | AddToughness$ 1 | EffectZone$ Command | Affected$ Artifact.YouCtrl | AffectedZone$ Battlefield | Description$ Artifact creatures you control get +1/+1.
+SVar:ReduceCost:Mode$ ReduceCost | ValidCard$ Artifact | Type$ Spell | Activator$ You | Amount$ 1 | Description$ Artifact spells you cast cost {1} less to cast. | Description$ Artifact spells you cast cost 1 less to cast.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/rebalanced/a-saheeli_filigree_master.txt:10]: 2x `Description$`

File: forge-gui/res/cardsfolder/rebalanced/a-soul_of_windgrace.txt [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+T:Mode$ Attacks | ValidCard$ Card.Self | OptionalDecider$ You | Execute$ TrigChangeZone | Secondary$ True | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, you may put a land card from a graveyard onto the battlefield tapped under your control.
+SVar:TrigChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | GainControl$ True | Tapped$ True | Hidden$ True | ChangeType$ Land
+A:AB$ GainLife | Cost$ G Discard<1/Land> | LifeAmount$ 3 | SpellDescription$ You gain 3 life.
+A:AB$ Draw | Cost$ R Discard<1/Land> | SpellDescription$ Draw a card.
+A:AB$ Pump | Cost$ B Discard<1/Land> | KW$ Indestructible | SubAbility$ DBTap | StackDescription$ SpellDescription | SpellDescription$ CARDNAME gains indestructible until end of turn.
+SVar:DBTap:DB$ Tap | Defined$ Self | StackDescription$ SpellDescription | SpellDescription$ Tap it.
+DeckHas:Ability$LifeGain|Discard & Keyword$Indestructible
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/rebalanced/a-soul_of_windgrace.txt:12]: this should really have `Graveyard` Ability too

**Review by Northmoc — APPROVED** [review]
Nice work!

**Pull request merged**